.github/workflows/test-helm-chart-release.yml

```diff
@@ -0,0 +1,50 @@
+name: Lint and Test Charts
```

Comment:
NOTE to self: Before merging this, make sure that all projects set the values that are NOT required anymore